src/library/Cardano/CoinSelection/Algorithm/Migration.hs

```diff
@@ -82,28 +62,42 @@ import qualified Internal.Coin as C
 -- Coin Selection for Migration
 --------------------------------------------------------------------------------
 
--- | Construct a list of coin selections / transactions to transfer the totality
--- of a user's wallet. The resulting 'CoinSelection' do not contain any
--- 'outputs', but only change coins (so there's no restriction about how
--- addresses are generated).
+-- | Creates a __series__ of coin selections that, when published as
+--   transactions, will have the effect of migrating all funds from one
+--   wallet to another.
+--
+-- Since UTxO-based blockchains typically impose limits on the sizes of
+-- individual transactions, and since individual UTxO sets can contain
```

Comment:
`UTxO sets` ot `UTxO set` ?

Reply:
> `UTxO sets` ot `UTxO set` ?

This is correct, because we've written:

> "individual UTxO sets can contain arbitrarily many entries."

(The sentence is referring to the concept of UTxO sets in general.)

We could also have written:

> "an individual UTxO set can contain arbitrarily many entries."

(Either would be correct.)